libs/portals/admin/ids-admin/src/components/Client/Permissions.tsx

```diff
@@ -98,33 +122,34 @@ function Permissions({ data = mockData }: PermissionsProps) {
                 <T.HeadData>
                   {formatMessage(m.permissionsTableLabelDescription)}
                 </T.HeadData>
-                <T.HeadData>
-                  {formatMessage(m.permissionsTableLabelAPI)}
-                </T.HeadData>
                 <T.HeadData>{/* For matching column count */}</T.HeadData>
               </T.Row>
             </T.Head>
             <T.Body>
-              {data.map((item) => (
-                <T.Row key={item.id}>
+              {[...permissions].map((item) => (
```

Comment:
`permissions` is already an array
```suggestion
{permissions.map((item) => (
```